packages/office-ui-fabric-react/src/components/Checkbox/__snapshots__/Checkbox.test.tsx.snap

```diff
@@ -94,11 +95,25 @@ exports[`Checkbox renders Checkbox correctly 1`] = `
             justify-content: center;
             margin-right: 4px;
             overflow: hidden;
+            position: relative;
             transition-duration: 200ms;
             transition-property: background, border, border-color;
             transition-timing-function: cubic-bezier(.4, 0, .23, 1);
             width: 20px;
           }
+          &:after {
```

Comment:
it seems weird that we're adding this rule without any modifications. Can we only define the after rule when we need it?